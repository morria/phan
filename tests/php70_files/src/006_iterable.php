<?php

function test_iterable_param(iterable $x) {
    var_export($x);
}
function test_iterable_return() : iterable {
    return [];
}
test_iterable_param([]);
var_export(test_iterable_return());
