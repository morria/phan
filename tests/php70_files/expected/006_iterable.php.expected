%s:3 PhanCompatibleIterableTypePHP70 Return type 'iterable' means a Traversable/array value starting in PHP 7.1. In PHP 7.0, iterable refers to a class/interface with the name 'iterable'
%s:6 PhanCompatibleIterableTypePHP70 Return type 'iterable' means a Traversable/array value starting in PHP 7.1. In PHP 7.0, iterable refers to a class/interface with the name 'iterable'

